src/__tests__/deployments.helper.spec.ts

```diff
@@ -0,0 +1,125 @@
+import { getDeploymentVersionsByChainIds } from '@/__tests__/deployments.helper';
+
+describe('Deployments helper', () => {
+  describe('getDeploymentVersionsByChainIds', () => {
+    it('should return all CompatibilityFallbackHandler versions for the specified chain ID', () => {
+      const versions = getDeploymentVersionsByChainIds(
```

Comment:
For each test, we parse a JSON file `N` times (where `N` is the number of asset files which does not change). My suggestion is to do an initial test setup where we get the deployments object and then each test asserts the respective contents.

Reply:
I've kept this function as is but it now references a singleton map of contracts-chain-versions. See my [previous comment](https://github.com/safe-global/safe-client-gateway/pull/1195#discussion_r1502480176) for more information.